tests/topotests/ospf-sr-topo1/r1/ospf_srdb.json

```diff
@@ -112,10 +112,10 @@
         {
           "prefix":"10.0.255.1\/32",
           "sid":100,
-          "inputLabel":20100,
+          "inputLabel":0,
```

Comment:
I'm a bit irritated here why the input label changes :O

Reply:
If you look to r1 configuration, I change the prefix-sid configuration to use the new `explicit-null` flag. Thus, the input Label becomes 0. Previously, r1 use the `no-php-flag` which explains why the input label was set to 20100.

Reply:
Yea, I've seen that but to me this just explains the change for the _outputLabel_. I would have thought that the inputLabel stays the same. Probably this is something basic which I don't understand ;)